media/css/firefox/ios.less

```diff
@@ -624,7 +624,8 @@ a.go {
     top: -30px;
 }
 
-.send-to-device {
+.send-to-device,
+.send-to {
```

Comment:
Note: this doesn't have any dependency on the test passing/failing here. Just fixing a visual discrepancy with a cropped button:

![button-cropped](https://cloud.githubusercontent.com/assets/400117/12556254/f0eddda8-c37c-11e5-9537-932c97209c81.png)

I supposed I *could* add a test for this also, although I wouldn't consider it the main page CTA.